project/src/store/reduser.ts

```diff
@@ -25,6 +26,12 @@ const reducer = createReducer(initialState, (builder) => {
     .addCase(changeSorting, (state, action) => {
       state.sorting = action.payload;
       state.isOpenSort = !state.isOpenSort;
+    })
+    .addCase(loadOffers, (state, action) => {
+      state.offers = action.payload;
+    })
+    .addCase(setOffersDataLoadingStatus, (state, action) => {
+      state.isOffersDataLoading = action.payload;
```

Comment:
у асинк санков есть состояния (pending, fulfilled, rejected) и если ты их будешь обрабатывать не придётся отдельные экшены создавать, типа 
```suggestion
    .addCase(loadOffers, (state, action) => {
      state.offers = action.payload;
    })
    .addCase(loadOffers.fulfilled, (state, action) => {
      state.isOffersDataLoading = action.payload;
```